packages/core/src/application.ts

```diff
@@ -88,6 +91,9 @@ export class Application extends Context {
     response: ServerResponse,
   ) {
     this._setupHandlerIfNeeded();
+    if (request.method === 'GET' && request.url === '/openapi.json') {
+      return this._serveOpenApiSpec(request, response);
```

Comment:
I'm surprised you can't use the same methods from a sequence to do the route matching.

> if (request.method === 'GET' && request.url === '/openapi.json')

This just seems fragile. We'll certainly want to allow the url to be configurable and to support `yaml`.